app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 16.sp
+                )
+            )
+        }
+            IconButton(
+                onClick = addToFavorites,
+                modifier = Modifier
+                    .background(
+                    shape = CircleShape,
+                    color = MaterialTheme.colorScheme.onTertiary
+                )
+                    .size(dimensionResource(R.dimen.show_screen_app_bar_icon_button_size))
+            ) {
+                Icon(
+                    imageVector = Icons.Outlined.FavoriteBorder,
+                    tint = MaterialTheme.colorScheme.onPrimary,
+                    contentDescription = null,
+                    modifier = Modifier.size(dimensionResource(R.dimen.show_screen_app_bar_favorite_icon_size))
+                )
+            }
+    }
+}
+
+@Composable
+private fun ShowScreenCoverImage(
+    imagePath: String,
+    modifier: Modifier = Modifier
+){
+     val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to MaterialTheme.colorScheme.onBackground,
+                0.95f to MaterialTheme.colorScheme.surfaceContainer,
+            ),
+        )
+
+    Box(
+        modifier = modifier
+    ){
+        AsyncImage(
+            model = imagePath,
+            modifier = Modifier.fillMaxSize(),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            placeholder = painterResource(R.drawable.placeholder_image)
+        )
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(brush = brush)
+        )
+    }
+}
+
+@Composable
+private fun ShowScreenMediaBasicInfo(
+    title:String, date:String,
+    language:String, progress: Int,
+    duration: String?, seasons:Int?,
+    modifier: Modifier = Modifier
+) {
+
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Box(
+            contentAlignment = Alignment.Center
+        ) {
+            CircularProgressIndicator(
+                progress = { progress/100f},
+                modifier = Modifier.size(72.dp),
+                trackColor = MaterialTheme.colorScheme.secondaryContainer,
+                color = MaterialTheme.colorScheme.secondary,
+                strokeWidth = dimensionResource(R.dimen.circular_progress_indicator_stroke_width)
+            )
+            Text(
+                text = "$progress%",
+                style = MaterialTheme.typography.bodyMedium
+            )
+        }
+        Spacer(
+            modifier = Modifier.width(dimensionResource(R.dimen.show_screen_media_info_spacer_width_1))
+        )
+        Column(
+            modifier = Modifier.fillMaxWidth()
+        ) {
+            val year = getYearFromDateString(date)
```

Comment:
get year as a parameter, and not in the component composable

Reply:
done